app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,51 @@
+class QuestionsController < ApplicationController
+  before_action :find_test, only: %i[index new create]
+  before_action :find_question, only: %i[show destroy]
+
+  rescue_from ActiveRecord::RecordNotFound, with: :rescue_with_question_not_found
+
+  def index
+    @questions = @test.questions.all
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def show
+    render plain: @question.body
+  end
+
+  def create
+    @question = @test.questions.new(post_params)
+    if @question.save
+      redirect_to @test
+    else
+      render new
+    end
+  end
+
+  def destroy
+    @test = @question.test
+    @question.destroy
+    redirect_to @test
```

Comment:
```suggestion
    @question.destroy
    redirect_to @question.test
```
объект будет доступен в памяти после удаления из базы